arm/README.md

```diff
@@ -206,14 +205,14 @@ package main
 
 import(
   "fmt"
-  "os"
 
   "github.com/azure/azure-sdk-for-go/examples/helpers"
   "github.com/azure/azure-sdk-for-go/arm/storage"
 )
 
-func main() {
-  name := os.Args[1]
+func create_account(args []string) {
```

Comment:
I'm feeling like this func should take two separate string parameters not an arr?